data/mods/blazemod/vehicleparts/blob_frames.json

```diff
@@ -0,0 +1,203 @@
+[
```

Comment:
Frames, painstakingly separated out. This is where most of the tile pains will come from, but once done, it doesn't need to be done again. Unchanged otherwise.